docker/package/tezos_setup_wizard.py

```diff
@@ -14,17 +14,12 @@
 import urllib.request
 import json
 from typing import List
+from available_networks import networks
```

Comment:
Yeah, this would be nicer than sticking everything into one shared file. Unfortunately, this is not going to work in its current form. You can see #388 for a bit more context, but the gist of it is that we can't assume we can import stuff from files local to this repository, as the .deb package we build with docker behaves/installs things mostly very differently on our end users' machines.

If you are set on retaining this additional file, you need to also add to [`model.py`](https://github.com/serokell/tezos-packaging/blob/master/docker/package/model.py) (look how the rest of this wizard is handled there). But honestly, until #388 is solved, I'd prefer not to clutter the `/usr/bin/` with any more files than necessary, so I think it's better to just stick the networks (and anything else you factor out) into the one `wizard_structure.py`.